mpt/impl/managesieve/file/src/test/java/org/apache/james/mpt/managesieve/file/host/FileHostSystem.java

```diff
@@ -39,7 +39,7 @@ protected static SieveRepository createSieveRepository() throws Exception {
     }
 
     public FileHostSystem() throws Exception {
-        super(new MemoryUsersRepository(), createSieveRepository());
+        super(new MemoryUsersRepository(false), createSieveRepository());
```

Comment:
you don't help the reader to understand you don't want virtual domain support. Please consider a more explicit way to do this.